vuu-ui/cypress/e2e/layout-management/screenshot.cy.js

```diff
@@ -1,29 +1,25 @@
 import "cypress-iframe";
-import { SHELL_WITH_NEW_THEME_URL } from "../../support/e2e/constants";
+import { ShellWithNewTheme } from "../../pages/ShellWithNewTheme";
+import { SaveLayoutDialog } from "../../pages/SaveLayoutDialog";
+
+const page = new ShellWithNewTheme();
+const dialog = new SaveLayoutDialog();
 
 context("Screenshot", () => {
   beforeEach(() => {
-    cy.visit(SHELL_WITH_NEW_THEME_URL);
+    page.visit();
   });
 
   // TODO (#VUU24): Improve test alignment with the user flow
```

Comment:
Can we remove this comment now?

Reply:
Yeah we've got plenty of tickets to cover this off now.